samples/dashboards/demo/datacursor-sync/demo.js

```diff
@@ -40,12 +40,40 @@ Dashboards.board('container', {
 
 // Build chart options for each HighchartsComponent
 function buildChartOptions(type, table, cursor) {
+    const typeString = type.charAt(0).toUpperCase() + type.slice(1);
+
     return {
         chart: {
+            spacing: [20, 20, 0, 20],
+            margin: [50, 50, 80, 100],
             events: {
                 load: function () {
                     const chart = this;
                     const series = chart.series[0];
+                    console.log(series);
+                    if (series.userOptions.type === 'pie') {
+
+                        chart.update({
+                            chart: {
+                                margin: 50,
+                                spacing: 30
+                            },
+                            legend: {
+                                enabled: true,
+                                layout: 'vertical',
+                                align: 'center',
+                                x: -200,
+                                verticalAlign: 'middle'
+                            }
+                        });
```

Comment:
Replace `});` with `}, false)` to avoid double redraw.

Reply:
done